e2e/apprunner/front-end/main.go

```diff
@@ -64,36 +65,43 @@ type Secret struct {
 }
 
 // DBGet calls an aurora DB and returns a timestamp from the database.
-func DBGet(w http.ResponseWriter, req *http.Request, ps httprouter.Params) {
-	sess, err := session.NewSession(&aws.Config{
-		Region: aws.String(os.Getenv("AWS_DEFAULT_REGION")),
-	})
+func DBGet(w http.ResponseWriter, req *http.Request) {
+	ctx, cancel := context.WithTimeout(req.Context(), 20*time.Second)
+	defer cancel()
+
+	cfg, err := config.LoadDefaultConfig(ctx,
+		config.WithRegion(os.Getenv("AWS_DEFAULT_REGION")),
+	)
 	if err != nil {
-		log.Printf("🚨 initial new aws session: err=%s\n", err)
+		log.Printf("🚨 load aws config: %s", err)
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 		return
 	}
-	ssm := secretsmanager.New(sess)
-	out, err := ssm.GetSecretValue(&secretsmanager.GetSecretValueInput{
+
+	mgr := secretsmanager.NewFromConfig(cfg)
+	out, err := mgr.GetSecretValue(ctx, &secretsmanager.GetSecretValueInput{
 		SecretId: aws.String(os.Getenv("FRONTENDCLUSTER_SECRET_ARN")),
 	})
 	if err != nil {
 		log.Printf("🚨 retrieve aurora secret: err=%s\n", err)
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 		return
 	}
-	secretValue := aws.StringValue(out.SecretString)
+
+	secretValue := aws.ToString(out.SecretString)
```

Comment:
oh did they rename it 👀

Reply:
lol yeah, gotta save those 3 characters 💪📈